yarn.lock

```diff
@@ -7163,6 +7163,15 @@ create-jest-runner@^0.5.3:
     jest-worker "^24.0.0"
     throat "^4.1.0"
 
+create-react-class@^15.6.0:
```

Comment:
Hmm why do we get this dependency again? 🤔

Reply:
I have no idea. Flopflip doesn't use it. I just ran changed the flopflip versions. I would not manually fiddle with the lockfile. Was this causing issues?

Reply:
Just wondering...maybe try regenerating the lockfile.

Reply:
566b306

Reply:
<img width="953" alt="CleanShot 2019-08-30 at 09 49 07@2x" src="https://user-images.githubusercontent.com/1877073/64002904-746ead80-cb0b-11e9-9b4d-c5617ac9dced.png">